src/uu/test/src/parser.rs

```diff
@@ -204,57 +218,63 @@ impl Parser {
 
         match peek3.as_slice() {
             // case 1: lparen is a literal when followed by nothing
-            [] => self.literal(Symbol::LParen.into_literal()),
+            [] => {
+                self.literal(Symbol::LParen.into_literal())?;
+                Ok(())
+            }
 
             // case 2: error if end of stream is `( <any_token>`
-            [symbol] => {
-                show_error!("missing argument after ‘{:?}’", symbol);
-                std::process::exit(2);
```

Comment:
Is this exit code now covered in `test.rs`? Is there a test for this so we're sure the exit code is still 2?

Reply:
So this PR is a draft?